File: week-02/day-02/diagonal_matrix.py
# - Create (dynamically) a two dimensional list
#   with the following matrix. Use a loop!
#
#   1 0 0 0
#   0 1 0 0
#   0 0 1 0
#   0 0 0 1
#
# - Print this two dimensional list to the output
oneline = ""

for i in range(0,4):
    oneline=""
    for j in range(0,4):
        if i==j:
            oneline += "1"
        else:
            oneline +="0"
        oneline+=" "
    print(oneline)

matrix = []
size = 3
for n in range(size):
    row = []
    for m in range(size):
        if n==m:
            row.append("1")
        else: 
            row.append("0")
    matrix.append(row)

for matrix_row in matrix:
    print(matrix_row)

File: week-08/day-02/Functions/matrix/diagonal_matrix.js
'use strict';
// - Create (dynamically*) a two dimensional list
//   with the following matrix**. Use a loop!
//
//   0 0 0 1
//   0 0 1 0
//   0 1 0 0
//   1 0 0 0
//
// - Print this two dimensional list to the console
//
// * size should depend on a variable
// ** Relax, a matrix is just like an array

var size = 5;

function drawMatrix(size_number){
    var line = "";
    var size = size_number;
    for(var i = 1; i<=size;i++){
        for(var j=1;j<=size;j++){
            if(j===size_number){
                line+="1"
            }else{
                line+="0"
            }
        }
        size_number--;
        console.log(line)
        line = ""
    }
}

drawMatrix(size)
